stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/list/ChannelItem.kt

```diff
@@ -0,0 +1,246 @@
+package io.getstream.chat.android.compose.ui.channels.list
```

Comment:
Leading Composable is called `DefaultChannelItem` in a file named `ChannelItem.kt`.

Might lead to slightly poorer discoverability.